blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxMetricsWriter.java

```diff
@@ -34,7 +34,7 @@ public void insertFullMetrics(Collection<Metric> metrics) throws IOException {
     }
 
     @Override
-    public void insertPreaggreatedMetrics(Collection<IMetric> metrics) throws IOException {
+    public void insertPreaggreatedMetrics(Collection<IMetric> metrics) throws Exception {
```

Comment:
What code changes in this class is throwing more exceptions other than IOException that cause you to change this throws definition?